packages/react-components/src/AddressMini.tsx

```diff
@@ -67,6 +68,7 @@ function AddressMini ({ balance, bonded, children, className, iconInfo, isPadded
                 <AccountName
                   noLookup={noLookup}
                   value={value}
+                  withSidebar={withSidebar}
```

Comment:
It is probably cool if these display an underline (maybe dotted) on hover - will just aid discoverability at least at the start.